config/nrfconnect/chip-module/Kconfig

```diff
@@ -284,4 +280,11 @@ config CHIP_ENABLE_READ_CLIENT
 	  This config can be disabled for device types that do not require Read Client functionality.
 	  Disabling this config can save flash and RAM space.
 
+config CHIP_CRYPTO_PSA_ITS_NVM_OFFSET
+	hex "Specify the offset for Matter crypto keys in the ITS NVM space"
+	default 0x30000
```

Comment:
Is it some constant or is it obvious how to get this offset? For me it would be nice to add at least one sentence of explanation in the help section on how to get it from some macro or something, as it looks like a magic number.

Reply:
Not sure if this is useful to have this exposed to Kconfig (it's probably us who need to make sure PSA key ID space is divided properly across components and customers usually won't be interested in changing that), but if you think otherwise please, remove "Specify the" part from the prompt to keep the style more consistent with others.